package.json

```diff
@@ -6,11 +6,13 @@
     "start": "http-server ./app -p 3000 -c-1",
     "test": "karma start ./test/karma.conf.js"
   },
-  "dependencies": {},
+  "dependencies": {
+    "jasmine": "^2.8.0"
```

Comment:
Do we need jasmine for production build?

Reply:
I installed it because I was trying to run the unit tests, so no

Reply:
Should I remove it?

Reply:
@carol-he u could move it to `dev-dependencies`